src/js/_enqueues/lib/emoji-loader.js

```diff
@@ -232,24 +232,24 @@
 				return ! isIdentical;
 			case 'emoji':
 				/*
-				 * Four and twenty blackbirds baked in a pie.
+				 * Person Running Facing Right: Medium Skin Tone
```

Comment:
We usually have some fun with this line as the description is below. With the change of emoji, I suggest

```suggestion
				 * Rise Like a Phoenix.
```